src/db/util/raw-model.ts

```diff
@@ -45,6 +45,8 @@ export type FindFn<F extends FieldDefinition, AdditionalArgs = {}> = (
      */
     skipValidation?: boolean;
     trx?: Knex.Transaction<any, any>;
+    orderByNullsLast?: boolean;
+    distinct?: Array<keyof InstanceDataOf<F>>;
```

Comment:
I'd document that this field uses `DISTINCT ON` clause, since it is specific to PostgreSQL and different from plain `DISTINCT`, which exists in SQL standard.
```suggestion
    /**
     * Uses `DISTINCT ON`, which is a feature unique to PostgreSQL.
     *
     * Keeps only the first row of each set of rows where the given (set of) column(s) is unique.
     * Note that the “first row” of each set is unpredictable unless `ORDER BY` is used to
     * ensure that the desired row appears first.
     */
    distinct?: Array<keyof InstanceDataOf<F>>;
```